src/identity.js

```diff
@@ -1786,27 +1690,27 @@ export default function Identity(mpInstance) {
 
     this.sendUserIdentityChangeEvent = function(
         newUserIdentities,
-        method,
         mpid,
+
+        // Mislabeled, could be previous or incoming
```

Comment:
Let's remove this comment and update the variable name to something more accurate